Maple2.Database/Storage/Game/GameStorage.Item.cs

```diff
@@ -98,6 +98,8 @@ public partial class Request {
             return (info.Meso, info.Expand);
         }
 
+        public short GetHairStorageAmount(long characterId) => Context.CharacterConfig.Find(characterId)?.HairSlotAmount ?? 0;
```

Comment:
Won't need this after moving to `CharacterUnlock` since you can access via `player.Unlock.HairSlotAmount`